chicken-sloths/bangazon-cli-sprint2
# Adding Payment_Types table to our database

# Description
This PR adds a new table to our database called Payment_Types. The table is a one mug to many mugs relationship. 

## Related Ticket(s)
#3 

## Problem to Solve
This solves the issue of when we create a new payment option for a customer, they can select the payment type and it will save that payment type id to their payment option, which will reference the Payment_Types table. This way, we will have a sungle source of truth for our payment types. Oddly enough, our payment types "source of truth" is actually just an array that is in the _paymentTypesFaker.js file.

## Proposed Changes
Describe the proposed changes.

## Expected Behavior
It should be like really good now.

## Steps to Test Solution

1. run `npm run db:reset` (DEF do this (as hell) to make sure your local database has the new data in it)
1. run `npm start`
1. select `2` and activate a customer
1. select `3` and select a payment type and enter an account number
1. Open the database and check that the new payment type was entered correctly

## Testing


## Documentation
